onnxruntime/core/providers/openvino/ov_versions/capability.cc

```diff
@@ -146,12 +144,8 @@ std::vector<std::unique_ptr<ComputeCapability>> GetCapability::Execute() {
       }
 
       //If subgraph has less then three, graph is considered trivial
-      if (data_ops_->GetVersion() != V_2020_4) {
-
-        if (this_cluster.size() < 3) {
-          continue;
-        }
-
+      if (this_cluster.size() < 3) {
```

Comment:
remind me why this logic previously only applied for non- 2020.4   (so it used to apply for 2020.3 and 2021.1 ?)

Reply:
This logic is supposed be to be a common one for all the versions of OpenVINO. Whenever there's a cluster with less than 3 nodes, we reject it and make it fall back to default CPU.

I am not sure why sahar added this check only for non - 2020.4. This was added as part of getcapability() refactoring PR. But anyways, since 2020.4 is being removed now, I think we can get rid of that additional if check.

Now it applies for all OV versions.